python/lancedb/table.py

```diff
@@ -325,14 +367,14 @@ def checkout(self, version: int):
         >>> db = lancedb.connect("./.lancedb")
         >>> table = db.create_table("my_table", [{"vector": [1.1, 0.9], "type": "vector"}])
         >>> table.version
-        1
+        2
         >>> table.to_pandas()
                vector    type
         0  [1.1, 0.9]  vector
         >>> table.add([{"vector": [0.5, 0.2], "type": "vector"}])
         >>> table.version
-        2
-        >>> table.checkout(1)
+        3
+        >>> table.checkout(2)
```

Comment:
Yeah I did a slight refactor if LanceTable.create so that the version history looks the same regardless of whether you added data in the beginning or not. Also this way the schema/metadata manipulation paths are more consolidated